### ISSUE TITLE
Positive lookbehind not working?

### ISSUE THREAD

**Issue by jonas-w** [einaregilsson/Redirector#313]
https://regexfiddler.com/e/hpodexqrqlqj

all the tests do pass but when inserting this into redirector it tells me that the expected result is not matching.

![image](https://user-images.githubusercontent.com/32615971/181852230-3f9a51eb-f74d-4723-927e-7f175830a1fc.png)

**Comment by Gitoffthelawn** [comment]
Hi Jonas.

For `pattern type`, you have `wildcard` selected instead of `regular expression`.  To use a regex in the include pattern, you will need to select `regular expression` for the `pattern type`.

**Comment by jonas-w** [comment]
Makes sense. But when i switch it to regular expression it accepts everything i type into the example url which seems odd?
